tests/integration_test.py

```diff
@@ -246,30 +247,80 @@ def runTest(self):
         self.assertFalse(inspect_data['VolumesRW'][mount_dest])
 
 
-@unittest.skipIf(NOT_ON_HOST, 'Tests running inside a container; no syslog')
-class TestCreateContainerWithLogConfig(BaseTestCase):
-    def runTest(self):
-        config = docker.utils.LogConfig(
-            type=docker.utils.LogConfig.types.SYSLOG,
-            config={'key1': 'val1'}
+class CreateContainerWithLogConfigTest(BaseTestCase):
```

Comment:
While it's not consistent, I think @mnowster's changes are improvements, so I'm fine with it. Would be good to get the rest of the suite updated along similar lines.